README.md

```diff
@@ -164,8 +165,9 @@ popd
 ```
 
 ## Set the kubeconfig after a successful installation
+NOTE: By default, The `kubeconfig` file is generated in the repository root directory and follows the convention `kubeconfig.<cluster-name>`:
 ```bash
-export KUBECONFIG=/root/kubeconfig.vm
+export KUBECONFIG=/root/cluster-deployment-automation/kubeconfig.vm
```

Comment:
Isn't it generated in `/root` anymore?

Reply:
That is true, I didn't realize the cwd was root when the path is set for downloaded_kubeconfig_path, Will change it back!

Reply:
You can check the default behavior here: https://github.com/bn222/cluster-deployment-automation/blob/main/clustersConfig.py#L197

Reply:
Thank you Sal! In the code it says that by default, the kubeconfig file is saved in the current working directory unless you specify a different location in the config file. That matches the behavior of when I deployed without setting the kubeconfig option in the config, and it put the file in the root directory of the repository. Should I keep the old line and just specify in the note that by default it would be in the cwd unless specifed instead?

Reply:
Yes this makes sense to me

Reply:
Amended the commit with the changes!